tests/app/conftest.py

```diff
@@ -25,26 +26,53 @@ def voila_args_extra():
 @pytest.fixture
 def voila_config_file_paths_arg():
     # we don't want the tests to use any configuration on the system
-    return '--VoilaTest.config_file_paths=[]'
+    return '--Voila.config_file_paths=[]'
 
 
 @pytest.fixture
 def voila_args(voila_notebook, voila_args_extra, voila_config_file_paths_arg):
-    debug_args = ['--VoilaTest.log_level=DEBUG'] if os.environ.get('VOILA_TEST_DEBUG', False) else []
-    return [voila_notebook, voila_config_file_paths_arg] + voila_args_extra + debug_args
+    return [voila_notebook, voila_config_file_paths_arg] + voila_args_extra
 
 
 @pytest.fixture
-def voila_app(voila_args, voila_config):
-    voila_app = VoilaTest.instance()
-    voila_app.initialize(voila_args + ['--no-browser'])
-    voila_config(voila_app)
-    voila_app.start()
+def voila_app(server_config, voila_config, voila_args):
```

Comment:
I released `jupyter_server` 0.2.1 (see [changelog](https://github.com/jupyter/jupyter_server/blob/master/CHANGELOG.md)) today, which includes a pytest-plugin for jupyter_server. 

Inside this plugin, we've added a `serverapp` fixture and a `configurable_serverapp` fixture that handle the creation + teardown of a jupyter server instance between each test so you don't have to.

You can see an example using this plugin [here in nbclassic](https://github.com/Zsailer/nbclassic/blob/notebook-handlers/tests/conftest.py).